src/usr/share/gpgmailer/gpgmailmessage.py

```diff
@@ -94,6 +90,18 @@ def set_body(self, body):
         self._check_if_saved()
         self.message['body'] = body
 
+    # Adds a list of recipients.
+    #
+    # recipients: a list of one or more recipients for the message.
+    def set_recipients(self, recipients):
+        self._check_if_saved()
+        self.message['recipients'] = recipients
+
+    # Adds a list of gpg key fingerprints for recipients of the message.
+    def set_recipient_keys(self, recipient_keys):
+        self._check_if_saved()
+        self.message['recipient_keys'] = recipient_keys
```

Comment:
As we discussed last week, we should decide if sending applications should specify the recipient keys and e-mail addresses or just the e-mail addresses.

Reply:
Applications should specify only emails
But emails have to match a key in gpgmailer
only encrypt with a key that matches the email address
(maintain dicts by key and by email)